views/024_integrations.sql

```diff
@@ -0,0 +1,4 @@
+CREATE OR REPLACE VIEW integrations_list AS
+SELECT id, name, description, 'scrapers' AS integration_type, source, agent_id, created_at, updated_at, deleted_at, created_by FROM config_scrapers_with_status UNION
```

Comment:
We need all the job_history columns as well

Can we also add a test that calls the view ?